e2e/specs/expander_state.spec.js

```diff
@@ -26,15 +26,16 @@ describe("expandable state", () => {
 
     cy.get("[data-testid='stExpander']").should("contain.text", "b0_write");
 
+    const expander0 = "b0_expanderb0_write"
+    const expander1 = "b1_expanderb1_write"
     cy.getIndexed(".stButton button", 1).click();
-
     cy.get("[data-testid='stExpander']").should(
-      "not.contain.text",
-      "b0_write"
+      "contain.text",
+      expander1
```

Comment:
Do you know the reason for this change?

Reply:
@LukasMasuch it's because elements of `expander` are now rendered all at once, instead rerender after each click, so the previous Cypress `have.not` check didn't match